tests/models/test_queryparams.py

```diff
@@ -89,7 +89,7 @@ def test_empty_query_params():
 
 def test_invalid_query_params():
     with pytest.raises(
-        TypeError, match=r"Expected str, int, float, bool, or None\. Got 'bytes'\."
+        TypeError, match=r"Expected str, int, float, bool, or None. Got 'bytes'."
```

Comment:
You don't need the `r` if you aren't escaping the periods — this is doing a `re.search` so technically the `.` will be a wildcard but 🤷‍♀️ it's not likely to cause you any problems. You could use `re.escape` if you really wanted to be correct.